src/Appwrite/Utopia/Response/Model/UsageUsers.php

```diff
@@ -5,72 +5,61 @@
 use Appwrite\Utopia\Response;
 use Appwrite\Utopia\Response\Model;
 
+/**
+ * Class UsageUsers
+ *
+ * Model class representing a usage statistics for users and sessions.
+ */
 class UsageUsers extends Model
 {
+    /**
+     * Constant representing the type string.
+     *
+     * @var string
+     */
+    const TYPE_STRING = 'string';
+    const TYPE_METRIC = 'metric';
```

Comment:
We don't do this in other models, and it doesn't look like this is used in many places, so I don't think it's worth it to add.